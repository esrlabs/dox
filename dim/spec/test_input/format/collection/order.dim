document: ordered

enclosed:
    - order.dim
    - order.dim.expected

REQ1:
    verification_criteria: VC
    refs: REQ2
    asil: ASIL_B
    tester: me
    status: valid
    sources: x.y
    miscellaneous: MISC
    developer: you
    text: Do it now!
    type: information
    tags: memory
    text_french: fr
    text_german: gr
    verification_methods: on_target
    review_status: unclear
    verification_criteria_german: vgr
    comment_german: cgr
    comment_italian: igr
    comment: Oh well.
    cal: QM
    verification_criteria_french: vfr

metadata:    V1

REQ2:

