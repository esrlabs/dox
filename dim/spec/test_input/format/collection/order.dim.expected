document: ordered

metadata: V1

REQ1:
  type: information
  text:                     Do it now!
  text_french:              fr
  text_german:              gr
  verification_criteria:        VC
  verification_criteria_french: vfr
  verification_criteria_german: vgr
  tags: memory
  asil: ASIL_B
  cal: QM
  developer: you
  tester: me
  verification_methods: on_target
  status: valid
  review_status: unclear
  comment: Oh well.
  comment_german: cgr
  comment_italian: igr
  miscellaneous: MISC
  sources: x.y
  refs: REQ2

REQ2:

enclosed:
  - order.dim
  - order.dim.expected
